README.md

```diff
@@ -126,3 +126,9 @@ Linuxbrew is maintained by [Shaun Jackman](http://sjackman.ca), [Bob W. Hogg](ht
 ## License
 Code is under the [BSD 2-clause "Simplified" License](https://github.com/Linuxbrew/brew/tree/master/LICENSE.txt).
 Documentation is under the [Creative Commons Attribution license](https://creativecommons.org/licenses/by/4.0/).
+
+## Donate
+
+Linuxbrew is a non-profit project run entirely by unpaid volunteers. We need your funds to pay for continuous integration and the computer resources used to build precompiled binary bottles of your favourite formulae. Every donation will be spent on making Linuxbrew better for our users. Please consider [donating regularly to Linuxbrew through Patreon](https://www.patreon.com/linuxbrew). We appreciate your support and contribution, no matter the level.
+
+[Linuxbrew](http://linuxbrew.sh) is a fork of [Homebrew](http://brew.sh), the macOS package manager, for Linux. If you use Homebrew on macOS consider [donating to Homebrew on Patreon](https://www.patreon.com/homebrew).
```

Comment:
suggest "also consider" to imply that donating to Linuxbrew and Homebrew are not mutually exclusive :wink: